assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java

```diff
@@ -309,8 +309,6 @@ protected void configure() {
     if (Boolean.valueOf(System.getenv("CHE_METRICS_ENABLED"))) {
       install(new org.eclipse.che.core.metrics.MetricsModule());
       install(new InfrastructureMetricsModule());
-    } else {
-      install(new org.eclipse.che.core.metrics.NoopMetricsModule());
     }
```

Comment:
@vinokurig is it safe to remove the `.NoopMetricsModule` ? what would happen if `CHE_METRICS_ENABLED` is not set?

Reply:
>is it safe to remove the .NoopMetricsModule 

Yes, this was needed to mock the `OkHttpMetricsEventListenerProvider`, but we removed it as discussed [here](https://github.com/eclipse-che/che-server/pull/774#discussion_r1983249763)
> what would happen if CHE_METRICS_ENABLED is not set?

The metrics modules will not be installed. Since we removed the `OkHttpMetricsEventListenerProvider` we do not need to install the mock implementation for it.

Reply:
@vinokurig what I'm not clear is why that listener was added at the first place

Reply:
The `Che Server JVM` dashboard works as expected